src-ghc/Pact/GasModel/GasTests.hs

```diff
@@ -546,6 +547,17 @@ enforceKeysetTests = tests
       updateEnvMsgSig
       [enforceKeysetExpr]
 
+enforceSessionTests :: NativeDefName -> GasUnitTests
+enforceSessionTests = tests
+  where
+    enforceSessionExpr = defPactExpression [text| (enforce-session '$sampleLoadedKeysetName) |]
+    updateEnvMsgSession = setEnv (set eeSessionSig (listToMaybe $ F.toList samplePubKeysWithCaps))
+
+    tests =
+      createGasUnitTests
+      updateEnvMsgSession
```

Comment:
```suggestion
        updateEnvMsgSession
```

Reply:
Why the indentation change? All the calls to `createGasUnitTest` in this file seem formatted like:

```
    tests =
      createGasUnitTests
      arg1
      arg2
      arg3
```